src/generate/rust.rs

```diff
@@ -858,24 +838,24 @@ where
     let rust_slice_ty = Pat::rust_slice_ty();
     quote!(
         impl<'_a, I> #ident<'_a, 'static, I>
-            where I: ::gll::runtime::Input<Slice = #rust_slice_ty>,
+            where I: crate::gll::runtime::Input<Slice = #rust_slice_ty>,
```

Comment:
Oh noes, this is baaad. Can we do the PR in two steps? First, update the generator, and then the generated code?
Or at least I'm curious what errors we'd get, if we change the edition but add `#[allow(rust_2018_idioms)]` on `mod parse_grammar;`.